src/modules/map/components/openlayers/OpenLayersInternalLayer.vue

```diff
@@ -89,13 +90,23 @@ function shouldAggregateSubLayerBeVisible(subLayer) {
             :z-index="zIndex"
         />
         <div v-if="layerConfig.type === LayerTypes.GROUP">
-            <OpenLayersInternalLayer
-                v-for="(layer, index) in layerConfig.layers"
-                :key="`${layer.id}-${index}`"
-                :layer-config="layer"
-                :parent-layer-opacity="layerConfig.opacity"
-                :z-index="zIndex + index"
-            />
+            <!-- If the layer is a group but it is a WMS layer, we use OpenLayersWMSLayer directly so the WMS request is handled on the layer group level, reducing the number of request -->
+            <div v-if="layerConfig instanceof ExternalWMSLayer">
```

Comment:
that won't catch anything, as if above we check for `layerConfig.type === LayerTypes.GROUP`, this will always be an instance of `ExternalGroupOfLayers`

Reply:
@pakb I compared it with `ExternalWMSLayer` not `ExternalGroupOfLayers`, which are different. Or, am I missing something here?